templates/skeleton/app/routes/account_.authorize.tsx

```diff
@@ -1,5 +1,5 @@
-import type {LoaderFunctionArgs} from '@shopify/remix-oxygen';
+import {unstable_defineLoader as defineLoader} from '@shopify/remix-oxygen';
 
-export async function loader({context}: LoaderFunctionArgs) {
-  return context.customerAccount.authorize();
-}
+export const loader = defineLoader(async ({context, response}) => {
```

Comment:
That's confusing...

Reply:
super confusing indeed...

So the reason why I changed it to defineLoader is because the type for response in loader is `ResponseStub | undefined`., and I want to ensure response is always defined. 

Switching back to `loader` means I likely need to handle the case where ResponseStub is undefined. Or I can just assume ResponseStub is always defined. Which means this route is always being hit with Remix APIs. But at this point, I can also assume this is NOT a true Resource Route that would cause type error thou.

Reply:
In general I feel like loaders are more confusing with single fetch. Resource route loaders still return `new Response`, other loaders don't :(

Reply:
It seems like it doesn't matter if we wrap it in `defineLoader`, so maybe we just keep it consistent even if they say it's not recommended? 🤷🏼

Reply:
I am going with `loader` unless the type fail and I specifically needed `defineLoader` so far. Will see if anything fail thou. I am testing CA API right now and finding some interesting failure. Fixing it while testing to see slowly what the pattern is.